theoriq/extra/flask/v1alpha2/subscribe.py

```diff
@@ -0,0 +1,25 @@
+import logging
+import threading
+import time
+from typing import Callable
+
+from theoriq.api.v1alpha2.protocol.protocol_client import ProtocolClient
+from theoriq.biscuit import TheoriqBiscuit
+
+logger = logging.getLogger(__name__)
+
+
+def theoriq_subscribe_to_agent(agent_id: str, subscribe_fn: Callable[[str], None], access_token: str):
```

Comment:
return type annotation is missing. Might be something like that.
```suggestion
def theoriq_subscribe_to_agent(agent_id: str, subscribe_fn: Callable[[str], None], access_token: str) -> threading.Thread:
```